admin/app/components/solidus_admin/ui/table/component.html.erb

```diff
@@ -32,6 +32,7 @@
           <input
             name="q[<%= @search_key %>]"
             value="<%= params.dig(:q, @search_key) %>"
+            type="search"
```

Comment:
Type `search` was not used because it showed a double clear button

Safari: 
<img width="147" alt="image" src="https://github.com/solidusio/solidus/assets/1051/fd7b7c5c-9b98-4c6f-8bca-fcfee7fba60d">

Chrome: 
<img width="128" alt="image" src="https://github.com/solidusio/solidus/assets/1051/97fdc6ba-5271-4c40-a6da-de44fee48e74">


If we use it (and it would be best to do so) we should overcome that: https://css-tricks.com/webkit-html5-search-inputs/#aa-styling-search-graphical-widgets


Probably something like: 

```html
<input class="[&::-webkit-search-cancel-button]:appareance-none …">
```

Reply:
Oh, thanks for pointing that out. Fixed. I needed to add a new Tailwind variant to select the "pseudo-element".